packages/bpk-component-breadcrumb/README.md

```diff
@@ -29,6 +29,11 @@ export default class App extends Component {
 }
 
 ```
+
+## Schema Markup
+
+JSON-LD schema mark up can be added to improve the SEO of the component through the `schemaMetaData` property
```

Comment:
```suggestion
JSON-LD schema mark up can be added to improve the SEO of the component through the `schemaMetaData` property.
```
Also, should we make `JSON-LD` a link to a relevant resource for people to know what it is?

Reply:
Added the full stop and a link